### ISSUE TITLE
For data custom dataset, Loss function does not converge.

### ISSUE THREAD

**Issue by choibigo** [nuclearboy95/Anomaly-Detection-PatchSVDD-PyTorch#12]
I succeeded in learning with MVTech data.
After that, I want to learn with custom data.
I created a file structure like MVTech to train custom data.
I'm currently training wall images.
I'm going to learn 500 images of the wall.
**python main_train.py --obj=wallcrack --lr=1e-4 --lambda_value=1e-3 --D=64 --epoch=500** used the command
However, the Loss value does not converge as shown below.
![image](https://user-images.githubusercontent.com/38881179/114668605-e543f700-9d3b-11eb-8bc8-0a4b4c2c9686.png)

I would appreciate it if you could give me a Hyperparameter or some tips.

The picture below is an example of the image I want to learn.
Example) 
![image](https://user-images.githubusercontent.com/38881179/114667982-1ff95f80-9d3b-11eb-8062-3fc10987908d.png)
![image](https://user-images.githubusercontent.com/38881179/114667992-2556aa00-9d3b-11eb-9387-080b86c9058a.png)
![image](https://user-images.githubusercontent.com/38881179/114667997-27206d80-9d3b-11eb-8885-70001d096e03.png)
![image](https://user-images.githubusercontent.com/38881179/114667999-28519a80-9d3b-11eb-9b7f-e55434174b8b.png)
![image](https://user-images.githubusercontent.com/38881179/114668005-2982c780-9d3b-11eb-9625-ffcad1fa645b.png)

**Comment by janerikvw** [comment]
In the paper they mention that a larger lambda is preferred for textures, whereas small lambda does well for objects. (Page 12 in the paper, under hyperparameters)
